simpledotorg/simple-server
# Add ReportingPatientBloodSugars

**Story card:** [sc-7793](https://app.shortcut.com/simpledotorg/story/7793/update-the-materialized-views-to-include-diabetes-outcomes)

## Because

We need to setup a reporting pipeline for Diabetes reports. This view is a required to calculate the patient's  state per month

## This addresses

Adds a materialized view to get the latest blood sugar measurement for a patient in every calendar month. 

## Test instructions

1. Checkout to the branch locally
2. Run `bundle exec rails db:migrate`
3. You should be able to see the `reporting_patient_blood_sugars` materialized view in postgres
4. Run `bundle exec rails db:rollback` to test the rollback
